src/core_plugins/kibana/public/dashboard/panel/panel.html

```diff
@@ -1,4 +1,11 @@
-<div class="panel panel-default" ng-switch on="panel.type" ng-if="savedObj || error">
+<div
+  class="panel panel-default"
+  ng-switch on="panel.type"
+  ng-if="savedObj || error"
+  reporting-item
```

Comment:
what do you think about defining this on <visualize> ?
- do we get any benefits from defining it here ?
- defining it on visualize would make it more consistent (on every page visualize needs to tell you when its done ... doesnt matter if its stand alone (editor), in dashboard panel, or somewhere else (monitoring ?)